sections/discussion/moscow-evaluation.tex

```diff
@@ -0,0 +1,72 @@
+\section{MoSCoW evaluation}
+Our MoSCoW was previously described in \autoref{sec:moscow}. 
+This section is an evaluation of the requirements we have in the MoSCoW.
+
+\subsection{Must have:}
+Almost all must have features have been completed.
+These were considered absolute essential to have a minimum viable product. 
+The only must have feature that we did not fully complete was testing for each component of the system to ensure quality.
+The different tests and test methods have previously been described in \autoref{chap:testing-the-system}.
+We did not have enough time to test everything as thoroughly as we wanted to
```

Comment:
måske evaluer på test ift API og front end separat

Reply:
^this

Reply:
api har nice tests